components/tag-list/tag-list.js

```diff
@@ -143,6 +160,23 @@ class TagList extends LocalizeCoreElement(ArrowKeysMixin(LitElement)) {
 		`;
 	}
 
+	updated(changedProperties) {
```

Comment:
I put this in updated rather than just in first update as `clearable` has the potential to change